Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/thoth-station/s2i-thoth-ubi8-py36
+
+USER 0
+
+RUN dnf install cloc -y
+RUN pip install --upgrade pip -y
+RUN pip install pipenv -y
+RUN pipenv shell
```

Comment:
Are these lines necessary? We don't want to ship pipenv in our container images. I would expect the following Dockerfile work:

```dockerfile
FROM quay.io/thoth-station/s2i-thoth-ubi8-py36

USER 0

RUN dnf install cloc -y --nodocs
USER 1001

```